### PR TITLE
Added Python tutorial at Tutorialspoint

### PR DESCRIPTION
**Pull request by Utkarsh299-tech**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -252,6 +252,7 @@
 * [Python for Everybody - Interactive](https://runestone.academy/runestone/books/published/py4e-int/index.html) - Barbara Ericson
 * [Python Koans](https://github.com/gregmalcolm/python_koans)
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
+* [Python Tutorial](https://www.tutorialspoint.com/python) - Tutorialspoint
```
Comment (SethiPandi) [inline free-programming-interactive-tutorials-en.md:255]: Please write nouns with the correct capitalization, "Tutorials Point" is two words in title case.

```suggestion
* [Python Tutorial](https://www.tutorialspoint.com/python) - Tutorials Point
```

**Review by SethiPandi — COMMENTED** [review]
Hey, and welcome to free programming books! :wave:

Just a small comment, it helps keep the repository more consistent and tidy.

**Comment by eshellman** [comment]
we've been adding tutorials point content in courses and in books. See #4573 and comment if you think they should go into interactive tutorials.

File: free-programming-interactive-tutorials-en.md (hunk 1 above)
Comment (Utkarsh299-tech) [reply free-programming-interactive-tutorials-en.md:255]: Okay sure, I will do that.

**Comment by eshellman** [comment]
based on your comments, add them in books instead.

**Comment by Utkarsh299-tech** [comment]
> based on your comments, add them in books instead.
Hi @eshellman, as you said I added them to books. I hope it's good now.

**Comment by Utkarsh299-tech** [comment]
Hi @eshellman thank you for merging this PR.
Can you add a 'hacktoberfest-accepted' label to it, please?

**Comment by SethiPandi** [comment]
> Hi @eshellman thank you for merging this PR.
> Can you add a 'hacktoberfest-accepted' label to it, please?

There is no need for the label.

PRs only need to be merged, approved **OR** get the label.

Plus Hacktoberfest should be over by now.

**Comment by Utkarsh299-tech** [comment]
> > Hi @eshellman thank you for merging this PR.
> > Can you add a 'hacktoberfest-accepted' label to it, please?
> 
> There is no need for the label.
> 
> PRs only need to be merged, approved **OR** get the label.
> 
> Plus Hacktoberfest should be over by now.

Okay, thank you for clearing my doubt.
And yes it's over now.

**Pull request merged**